src/aioftp/client.py

```diff
@@ -769,60 +915,73 @@ def list(self, path="", *, recursive=False, raw_command=None):
 
             >>> stats = await client.list()
         """
-
-        class AsyncLister(AsyncListerMixin):
-            stream = None
-
-            async def _new_stream(cls, local_path):
-                cls.path = local_path
-                cls.parse_line = self.parse_mlsx_line
+        ctx = self
```

Comment:
It is better to name it as `client` since the class provides client interface. Also, I suggest to pass `self` as argument to `AsyncLister`, so it will be `self.client`.

Reply:
Done